modules/primer-breadcrumb/lib/breadcrumb.scss

```diff
@@ -12,8 +12,13 @@
   }
 }
 
+nav ol:last-child .breadcrumb-item,
```

Comment:
This is too specific here, instead will you drop the `nav ol` and just leave `:last-child .breadcrumb-item`